src/components/Tooltip/Tooltip.module.scss

```diff
@@ -0,0 +1,56 @@
+.wrapper {
+  background: var(--color-core-neutral-darken-300);
+  color: var(--color-text-strong);
+  @include inner-border(var(--color-border-muted-alpha));
+
+  z-index: var(--z-index-tooltip);
+  overflow: hidden;
+  border-radius: var(--radii-xsmall);
+  padding: 6px 12px;
+  @include typography(text, 200, regular);
+  transition: var(--transition-fast);
+  will-change: transform, opacity;
+  user-select: none;
+
+  animation-name: enter;
+  animation-duration: 150ms;
```

Comment:
@toiletgranny what do you think about adding this duration to our tokens? Our `fast` felt definitely too slow for this animation. Should we create something like `super-fast`?

Reply:
I think the reason it felt slow was because of the lack of a animation timing function definition. I used `var(--motion-timing-fast)` (`300ms`) for the duration, and `var(--transition-function)` for the timing function and the end result feels a lot like the `150ms` imo. See for yourself and let me know if you like it, or if you prefer to add `150ms` as a new token.

Reply:
I have actually also used the proper timing function. However what makes the difference now is that the tooltip is a bit bigger (as the text is also bigger). It's looks super nice 😄